packages/next/src/build/templates/pages.ts

```diff
@@ -289,9 +289,8 @@ export async function handler(
                   reactLoadableManifest,
 
                   assetPrefix: nextConfig.assetPrefix,
-                  strictNextHead: Boolean(
-                    nextConfig.experimental.strictNextHead
-                  ),
+                  strictNextHead:
+                    nextConfig.experimental.strictNextHead ?? true,
```

Comment:
can we add strictNextHead: true to default config in config-shared.ts so we don't need to handle fallback when access it?

Reply:
we'll still need to have the fallbacks to make TypeScript happy (since all experimental properties are optional), but yeah I can update the default in there too